prow/plugins/size/gen_files_group_test.go

```diff
@@ -0,0 +1,319 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package size
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestGenFilesGroupLoad(t *testing.T) {
+	cases := []struct {
+		name      string
+		src       string
+		repoPaths []string
+		err       error
+
+		want GenFilesGroup
+	}{
+		{
+			name: "k8s-config",
+			src: `# Files that should be ignored by tools which do not want to consider generated
+# code.
+#
+# https://github.com/kubernetes/contrib/blob/master/mungegithub/mungers/size.go
```

Comment:
k should we file an issue about it in kubernetes/kubernetes [here](https://github.com/kubernetes/kubernetes/blob/master/.generated_files#L4)